src/integration-test/resources/application-test.properties

```diff
@@ -22,3 +22,9 @@ spring.main.banner_mode=off
 
 spring.datasource.username=root
 spring.datasource.password=
+
+# Using the Scheduler as Authentication Service
+pa.scheduler.rest.url=http://localhost:8080/rest
```

Comment:
If https is not served by Nginx (so if it's served by the release's Jetty) you'll need to modify this value. We cannot infer the value at runtime because we don't have a discovery mechanism that would allow us to know if it's http/https and the port.
Otherwise, it will work just fine (it's the case on our try* platforms).